cmake/process_users_input.cmake

```diff
@@ -6,11 +6,9 @@ set(GRAPH_PARTITIONING_LIB_OPTIONS parmetis parhip ptscotch)
 set_property(CACHE GRAPH_PARTITIONING_LIBS PROPERTY STRINGS ${GRAPH_PARTITIONING_LIB_OPTIONS})
 
 option(MPI "Use MPI parallelization" ON)
-option(MINI_SEISSOL "Use MiniSeisSol to compute node weights for load balancing" ON)
 option(OPENMP "Use OpenMP parallelization" ON)
 option(ASAGI "Use asagi for material input" OFF)
 option(MEMKIND "Use memkind library for hbw memory support" OFF)
-option(USE_IMPALA_JIT_LLVM "Use llvm version of impalajit" OFF)
```

Comment:
this is not really related to the PR?

Reply:
Not really; it's only an unused option at the moment IIRC—or at least, since easi was removed as a submodule.